cassdegrees/ui/views.py

```diff
@@ -207,6 +210,120 @@ def create_program(request):
 
     return render(request, 'createprogram.html', context=render_properties)
 
+
+def edit(request, element, program_id):
+    render_properties = {
+        'msg': None,
+        'is_error': False,
+    }
+
+    # api request requires ID as string
+    id_to_edit = str(program_id)
+
+    # Get the correct instance from the request and render form snippets with details
+    if element == "degree":
+        elementDesc = "Program Template"
+        model_object = DegreeModel.objects.get(id=program_id)
+        object_dict = model_to_dict(model_object)
+        form = EditProgramFormSnippet(initial=object_dict)
+        comment_snippet = StaffNotesFormSnippet(initial=object_dict)
+        content_snippet = ProgramContentSnippet(initial=object_dict)
+
+        model_api_url = 'http://127.0.0.1:8000/api/model/degree/'
+        # print(model_to_dict(model_object))
```

Comment:
Leftover print statements shouldn't be here

Reply:
Will definitely remove prior to merge, just waiting to see if anyone ran into bugs first.